tsdb/index/tsi1/log_file.go

```diff
@@ -1035,32 +1008,45 @@ type logFileMeasurementCompactInfo struct {
 	size   int64
 }
 
-// MergeMeasurementsSketches merges the measurement sketches belonging to this
-// LogFile into the provided sketches.
-//
-// MergeMeasurementsSketches is safe for concurrent use by multiple goroutines.
-func (f *LogFile) MergeMeasurementsSketches(sketch, tsketch estimator.Sketch) error {
+// MeasurementsSketches returns sketches for existing and tombstoned measurement names.
+func (f *LogFile) MeasurementsSketches() (sketch, tSketch estimator.Sketch, err error) {
 	f.mu.RLock()
 	defer f.mu.RUnlock()
+	return f.measurementsSketches()
```

Comment:
since you have named return variables there is no need to use a `defer` here.
It doesn't really matter but
```go
f.mu.RLock()
sketch, tSketch, err = f.measurementsSketches()
f.mu.RUnlock()
return sketch, tSketch, err
``` 
Would be a tiny bit more performant.  I doubt it would make much of a difference here, though.